styles.css

```diff
@@ -94,6 +95,304 @@ nav svg {
 }
 
 a :hover {
-  color: turquoise;
+  background-color: turquoise;
+  text-decoration: none;
+}
+
+.card {
+  display: grid;
+  padding: 16px;
+  grid-column: auto;
+  align-items: center;
+  gap: 12px;
+  border-radius: 16px;
+  border: 1px solid var(--neutrals-n-40, #dfe1e6);
+  background: var(--neutrals-n-0, #fff);
+  margin: 114px 24px 88px 24px;
+}
+
+.snap {
+  width: 295px;
+  height: 220px;
+  background-image: url("asset/images/snapshot-portfolio.png");
+  border-radius: 8px;
+}
+
+.card h2 {
+  display: grid;
+  width: 295px;
+  height: 36px;
+  grid-column: auto;
+  color: var(--neutrals-n-800, #172b4d);
+  font-family: Poppins, sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 44px;
+  margin-bottom: 12px;
+}
+
+.card p {
+  color: var(--neutrals-n-600, #344563);
+  font-family: Poppins, sans-serif;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.two p {
+  margin-top: 12px;
+  display: grid;
+  width: 295px;
+  height: 76px;
+  grid-column: auto;
+  color: var(--neutrals-n-600, #344563);
+  font-family: Poppins, sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.three {
+  display: flex;
+  grid-column: auto;
+  align-items: flex-start;
+  gap: 12px;
+  margin-top: 12px;
+}
+
+.three a {
+  padding: 4px 12px;
+  align-self: center;
+  border-radius: 8px;
+  background: var(--primary-b-50, #ebebff);
+}
+
+.four a {
+  display: flex;
+  padding: 12px;
+  align-items: center;
+  gap: 10px;
+  border-radius: 8px;
+  border: 1px solid var(--primary-b-400, #6070ff);
+  color: var(--primary-b-500, #396df2);
+  text-align: center;
+  font-family: Poppins, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.51px;
+  width: 128px;
+  height: 50px;
+}
+
+.svgclass a {
+  padding: 8px;
+}
+
+.three a:hover {
+  color: red;
+  text-decoration: none;
+}
+
+.four {
+  display: grid;
+  padding: 12px 169px 12px 0;
+  margin-top: 12px;
+  grid-column: auto;
+}
+
+.four a:hover {
+  color: red;
+  text-decoration: none;
+}
```

Comment:
- [ Required ] I think it would be better if you ensure that the `See Project` button follows the button interaction designs on the Figma template as requested by the previous reviewer. Currently, the styling applied on button hover is not consistent with the Figma design provided for this project. Please see the screenshot below for the button interaction styling to use.

## Example code
```CSS
.four a:hover {
  color: #fff;
  background-color: #6070FF;
  text-decoration: none;
}

.four a:active {
  color: #fff;
  background-color: #2230d2;
  text-decoration: none;
}

.four a:disabled {
  color: #c1c7d0;
  background-color: #fff;
  text-decoration: none;
  border: 1px solid #c1c7d0;
}
```
## Screenshot
![Screenshot from 2023-07-13 02-14-58](https://github.com/okayboy12/Portfolio-setup-and-mobile-first/assets/16392046/4a15893c-3ff9-4d3b-abf9-694ecadf0f84)